Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -12017,8 +12007,8 @@
     "Context": {
       "ReferenceDateTime": "2019-01-31T00:00:00"
     },
-    "Comment": "RELPERIOD. Refine support for relative periods.",
-    "NotSupported": "dotnet, javascript, python, java",
+    "Comment": "'miércoles pasado' seems to translate to 'last Wednesday' instead of 'past Wednesday'. Only 'este miércoles pasado' corresponds to 'this past Wednesday'.",
```

Comment:
We need to check this one with a fluent Spanish speaker. I know a bit, but I'm not native fluent.

Reply:
@tellarin, we asked a native Spanish speaker and he said:

The problem with Spanish is that we use “pasado” for both “last” and “past” and only context will help to disambiguate.
Some tips to help to avoid ambiguity are:
- Adding “este” -> to refer the day of the current week.
- Adding “de la semana pasada” -> to refer to the day of the previous week.